test/ClangImporter/objc_override.swift

```diff
@@ -116,6 +116,16 @@ class MyHashableNSObject: NSObject {
   }
 }
 
+// rdar://problem/47557376
+// Adding an override to someone else's class in an extension like this isn't
+// really sound, but it's allowed in Objective-C too.
+extension OverrideInExtensionSub {
+  open override func method() {}
+}
+public extension OverrideInExtensionSub {
+  open override func accessWarning() {} // expected-warning {{'open' modifier conflicts with extension's default access of 'public'}}
```

Comment:
@brentdax How does this sound? It's still vague about how it's going to be treated for now, but at least it sounds like a problem.

Reply:
Much better!